test/statement.test.ts

```diff
@@ -68,7 +68,7 @@ describe("statement", function () {
         (err: any) => {
           strictEqual(
             err.cause.message,
-            "error parsing statement: syntax error at position 36 near 'blurg'"
+            "error parsing statement: syntax error at position 36 near 'blurg'\n  CREATE TABLE test_run (counter blurg⚠);"
```

Comment:
@dtbuchholz I tried a few different formatting ideas, but having these errors be helpful to an application, and a terminal, and the different browser consoles is kinda restrictive.
What I landed on (so far) is inserting the unicode warning sign character.  This isn't the prettiest thing, but it's something you can find with your eyes or in code (with something like `indexOf`).
If you have any ideas or alternatives I can definitely explore those.

Reply:
@joewagner sorry for the delay! yes, i think this is already a great addition and would achieve what i was looking for.

the only thing i can think of that's different than ⚠ would be something like this:
```
error parsing statement: syntax error at position 36 near 'blurg'
CREATE TABLE test_run (counter blurg);
                               ^^^^^^
```
but I think that what you have is likely easier and already helpful enough—getting those `^` to line up can probs get messy.

Reply:
Hmm, I like the look of the carets more than the unicode symbol....
It will probably be kinda tricky, but I'll give it a try.

Reply:
@dtbuchholz changing to use the carrots was fairly easy.  Give this another look when you get a chance.